src/interactive/infoToggle.story.jsx

```diff
@@ -2,11 +2,50 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { decorateWithBasics } from '../utils/decorators';
 import { withInfo } from '@storybook/addon-info';
-import { withKnobs, text, object, select } from '@storybook/addon-knobs';
+import { withKnobs, text, object, select, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 
 import InfoToggle from './InfoToggle';
 
+class ManualInfoToggle extends React.PureComponent {
+	state = {
+		tooltipOpen: false,
+	};
+
+	toggleTooltip = () => {
+		this.setState(prevState => ({ tooltipOpen: !prevState.tooltipOpen }));
+	};
+
+	closeTooltip = () => {
+		this.setState(() => ({ tooltipOpen: false }));
```

Comment:
sort of marginal benefit here, but consider using composition rather than defining two separate methods that each update `state.tooltipOpen`

```js
changeTooltip(tooltipOpen) {
  this.setState({ tooltipOpen });
}
toggleTooltip = () => this.changeTooltip(!this.state.tooltipOpen)
closeTooltip = () => this.changeTooltip(false)
```